src/components/KPagination/KPagination.vue

```diff
@@ -203,7 +203,7 @@ const emit = defineEmits<{
 
 const currPage: Ref<number> = ref(props.currentPage ? props.currentPage : 1)
 const currentPageSize: Ref<number> = ref(props.initialPageSize ? props.initialPageSize : props.pageSizes[0])
-const pageCount: Ref<number> = ref(Math.ceil(props.totalCount / currentPageSize.value))
+const pageCount: ComputedRef<number> = computed((): number => Math.ceil(props.totalCount / currentPageSize.value))
```

Comment:
nit: this can be simplified
```suggestion
const pageCount = computed((): number => Math.ceil(props.totalCount / currentPageSize.value))
```